app/javascript/packs/base.jsx

```diff
@@ -95,6 +96,10 @@ window.Forem = {
       </Modal>,
       currentModalContainer,
     );
+
+    if (onOpen) {
```

Comment:
A callback won't always be supplied, hence we wrap it in an `if`

Reply:
Just seeing this now as I was on vacation. [Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) can be use instead of an existence guard.

```javascript
    if (onOpen) {
      onOpen();
    }
```

can be

```javascript
    onOpen?.();
```